<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalzarCarga</class>
 <widget class="QDialog" name="CalzarCarga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CARGAS PENDIENTES ORIGEN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedCompras">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableView" name="compras">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Cliente:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="cliente">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Cliente:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="tipo_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Tipo:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="tipo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Tipo:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Precio:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="precio">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Precio:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="moneda1">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Moneda:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="moneda">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Moneda:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="kilos1">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Kilos:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="kilos">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Kilos:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="modificar_compra">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string>Modificar</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="id_cliente">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CARGAS PEDIENTES DESTINO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedVentas">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="ventas">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Comprador:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="comprador">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>comprador</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Contrato:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="contrato">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Contrato</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="tipo_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Tipo:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="tipo1">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Tipo:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Precio:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="precio_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Precio:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="moneda1_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Moneda:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="moneda_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Moneda:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="kilos1_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Kilos:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="kilos_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Kilos:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="modificar_compra_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>26</height>
                </size>
               </property>
               <property name="text">
                <string>Modificar</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="comprador_id">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aceptar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelar">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
